.github/workflows/windows-conda.yml

```
@@ -0,0 +1,64 @@
+name: Build ndcurves for Windows via conda
+
+on: [push, pull_request]
```

```suggestion
on:
  push:
    branches:
      - devel
      - master
  pull_request:
    branches:
      - devel
      - master
```

This should avoid the workflow to be triggered twice